File: Java/1018 - Cedulas.java
import java.io.IOException;
import java.util.Scanner;


class Main {

	public static void Show_Info(int valor, int nota) {
		
		System.out.printf(valor + " nota(s) de R$ " + nota + ",00\n");

	}

	public static void main(String[] args) {

		Scanner input = new Scanner(System.in);

		int resto;

		int money = input.nextInt();
		System.out.println(money);

		int cem = money / 100;
		resto = money % 100;

		int cinq = resto / 50;
		resto = resto % 50;
		
		int vinte = resto / 20;
		resto = resto % 20;

		int dez = resto / 10;
		resto = resto % 10;

		int cinco = resto / 5;
		resto = resto % 5;

		int dois = resto / 2;
		resto = resto % 2;
	
		Show_Info(cem, 100);
		Show_Info(cinq, 50);
		Show_Info(vinte, 20);
		Show_Info(dez, 10);
		Show_Info(cinco, 5);
		Show_Info(dois, 2);
		Show_Info(resto, 1);
	}
}

File: CPP/1018 - Cedulas.cpp
#include <iostream>
#include <iomanip>

using namespace std;
int main() {

	int money;
	cin >> money;

	cout << money << endl;
	
	int cem = money/100;
	money = money%100;

	int cinq = money/50;
	money = money%50;

	int vinte = money/20;
	money = money%20;

	int dez = money/10;
	money = money%10;

	int cinc = money/5;
	money = money%5;

	int dois = money/2;
	int um = money%2;

	cout << cem << " nota(s) de R$ 100,00" << endl;
	cout << cinq << " nota(s) de R$ 50,00" << endl;
	cout << vinte << " nota(s) de R$ 20,00" << endl;
	cout << dez << " nota(s) de R$ 10,00" << endl;
	cout << cinc << " nota(s) de R$ 5,00" << endl;
	cout << dois << " nota(s) de R$ 2,00" << endl;
	cout << um << " nota(s) de R$ 1,00" << endl;
	return 0;
}